src/pages/migrating/guide/develop.mdx

```diff
@@ -79,199 +69,203 @@ To get started, uninstall the following packages if they exist in your project:
+[here](https://github.com/carbon-design-system/carbon/blob/main/packages/styles/docs/sass.md#configuration).
+We also have published a guide over on
+[Medium](https://medium.com/carbondesign/migrating-from-node-sass-to-sass-eba9db004a3a)
+to help out with common problems that come from this migration.
 
-```jsx
-// Before
-import { Accordion, AccordionItem } from 'carbon-components-react';
-// After
-import { Accordion, AccordionItem } from '@carbon/react';
-```
+### Step 3: Update style import paths
 
-### Step 7: Update icon imports
+Now, open your project's main stylesheet:
 
-<InlineNotification>
+- Any line of code with `@import` can be removed.
+- Any feature flags can be removed.
 
-This step can be automated using the `@carbon/upgrade` CLI. For full details,
-see the
-[`@carbon/upgrade`](https://github.com/carbon-design-system/carbon/tree/main/packages/upgrade)
-package documentation
+All you need is the following:
 
-</InlineNotification>
+```scss
+@use '@carbon/react';
+```
+
+When migrating your custom components, some of our most used sass assets are
+included below, along with what you would need to have at the top of your file
+to use it. Essentially, all we are doing is including the path where we would
+[find these styles](https://github.com/carbon-design-system/carbon/tree/main/packages/react/scss)
+in our package.
+
+| Carbon sass I'm using           | Sass modules to include                                  |
+| ------------------------------- | -------------------------------------------------------- |
+| Theme tokens                    | `@use '@carbon/react/scss/theme' as *`                   |
+| Theme mixins                    | `@use '@carbon/react/scss/themes' as *`                  |
+| Design language color tokens    | `@use '@carbon/react/scss/colors' as *`                  |
+| Spacing tokens                  | `@use '@carbon/react/scss/spacing' as *`                 |
+| Breakpoint mixins               | `@use '@carbon/react/scss/breakpoint' as *`              |
+| Motion tokens and functions     | `@use '@carbon/react/scss/motion' as *`                  |
+| Rem and other convert functions | `@use '@carbon/react/scss/utilities/convert' as *`       |
+| Z-index helper                  | `@use '@carbon/react/scss/utilities/z-index' as *`       |
+| Focus outline helper            | `@use '@carbon/react/scss/utilities/focus-outline' as *` |
+| Transform rotate helper         | `@use '@carbon/react/scss/utilities/rotate' as *`        |
+| Skeleton animation helper       | `@use '@carbon/react/scss/utilities/skeleton' as *`      |
+
+Now that you've migrated all of your imports, do a find/replace using your
+favorite code editor. Find `carbon--` and replace it with nothing. `carbon--` is
+no longer necessary. The full list of changes are available in our
+[Migration Docs](https://github.com/carbon-design-system/carbon/blob/main/docs/migration/v11.md#changing-import-paths-from-scssglobalsscss-to-scss).
+
```

Comment:
I might try re-wording this sentence, I had to re-read it a couple times to understand what you meant 😅 

```suggestion
Now that you've migrated all of your imports, do a find/replace using your
favorite code editor. The prefix`carbon--` is no longer necessary, you can 
do a find for `carbon--` and replace it with nothing to remove across all your files. 
The full list of changes are available in our
[Migration Docs](https://github.com/carbon-design-system/carbon/blob/main/docs/migration/v11.md#changing-import-paths-from-scssglobalsscss-to-scss).

```